src/lib/components/editor/TiptapRenderer.svelte

```diff
@@ -77,9 +77,14 @@
                 },
             },
             content: tiptapJson?.tiptap,
-            onTransaction: () => {
+            onTransaction: ({ editor }) => {
                 // force re-render so `editor.isActive` works as expected
                 editor = editor;
```

Comment:
I think this will break things because now `editor` is the locally bound variable and the Svelte part won't be reactive

Reply:
So, maybe:
```
  onTransaction: () => {
      // force re-render so `editor.isActive` works as expected
      editor = editor;
      
      if (editor) {
          onChange?.(
              editor.getJSON(),
              editor.storage.characterCount.words(),
              editor.storage.characterCount.characters()
          );
      }
  },
```
to avoid modifying the scope of `editor`.